src/lib/y2partitioner/widgets/menus/modify.rb

```diff
@@ -27,6 +27,13 @@ module Widgets
     module Menus
       # Class to represent the Modify/Edit menu
       class Modify < Device
+        # Constructor
+        def initialize(*args)
```

Comment:
Having constructor only because `textdomain` is annoying. I would extend the class with `Yast::I18n` and then we can call `textdomain` inside the class:

~~~
class Modify < Device
  extend Yast::I18n
  
  textdomain "storage"
~~~

Reply:
If it works, fine with me.